cognite/neat/core/workflow/cdf_store.py

```diff
@@ -43,7 +43,7 @@ def __init__(
         self.rules_storage_path = str(rules_storage_path)
         self.workflows_storage_type = "file"
 
-    def init_cdf_resources(self):
+    def init_cdf_resources(self, resource_type="all"):
```

Comment:
This parameter is not used. Is this intentional? If so, why is it there?

Reply:
fixed